android/README.md

```diff
@@ -130,7 +132,7 @@ public class MainActivity extends JitsiMeetActivity {
 ```
 
 Alternatively, you can use the `org.jitsi.meet.sdk.JitsiMeetView` class which
-extends `android.view.View`:
+extends `android.view.View` (note that this is not recommended):
```

Comment:
Why is this not recommended? And if it's not, why do we provide the option?

Reply:
Because it requires manually wiring the view to the activity with all that boilerplae code, most likely.

We ran into this in practice, that is, Atlassian was using a base Activity class which they needed all their activities to derive from, so using our Activity was not an option.

I consider this a valid case, but a rather advanced one, hence the "not recommended".

Reply:
Add something like that to the text! 

> Note that this should only be needed when the ```JitsiMeetActivity``` cannot be used for some reason. Extending ```JitsiMeetView``` requires manual wiring of  the view to the activity, using a lot of boilerplate code. Using the Activity instead of the View is often preferred.